Userland/Applications/FontEditor/FontEditor.cpp

```diff
@@ -293,26 +295,26 @@ FontEditorWidget::FontEditorWidget(const String& path, RefPtr<Gfx::BitmapFont>&&
     toolbar.add_action(*m_next_glyph_action);
     toolbar.add_action(*m_go_to_glyph_action);
 
-    m_scale_five_action = GUI::Action::create_checkable("500%", { Mod_Ctrl, Key_1 }, [&](auto&) {
-        m_glyph_editor_widget->set_scale(5);
-        m_glyph_editor_container->set_fixed_size(m_glyph_editor_widget->preferred_width(), m_glyph_editor_widget->preferred_height());
-        m_left_column_container->set_fixed_width(m_glyph_editor_widget->preferred_width());
+    i32 scale = Config::read_i32("FontEditor", "GlyphEditor", "Scale", 10);
+    m_glyph_editor_widget->set_scale(scale);
+
+    m_scale_five_action = GUI::Action::create_checkable("500%", { Mod_Ctrl, Key_1 }, [this](auto&) {
+        set_scale(5);
+        Config::write_i32("FontEditor", "GlyphEditor", "Scale", 5);
```

Comment:
Maybe you should move this to the set_scale function, same with the saving below.

Reply:
Fixed